libs/cardano-ledger-test/src/Test/Cardano/Ledger/Alonzo/Tools.hs

```diff
@@ -20,11 +20,11 @@ import Cardano.Ledger.Alonzo.TxWits
 import Cardano.Ledger.Alonzo.UTxO (AlonzoScriptsNeeded)
 import Cardano.Ledger.BaseTypes (ProtVer (..), ShelleyBase, natVersion)
 import Cardano.Ledger.Coin (Coin (..))
-import Cardano.Ledger.Core
-import qualified Cardano.Ledger.Crypto as CC
+import Cardano.Ledger.Crypto
```

Comment:
open import?

Reply:
Yes! In fact, I want to re-export `Crypto` type class from `Core`, so we don't have to import it in every single module in this repository. But that'll be done in a separate PR at some later point